pkgs/shells/rc/default.nix

```diff
@@ -17,6 +18,10 @@ stdenv.mkDerivation rec {
   buildInputs = [ ncurses ]
     ++ lib.optionals (readline != null) [ readline ];
 
+  preConfigure = ''
+    export CPPFLAGS="$CPPFLAGS -DSIGCLD=SIGCHLD"
```

Comment:
Would setting ``CPPFLAGS = [ "-DSIGCLD=SIGCHLD" ];`` overwrite some flags?

Reply:
Seems to work too.